public/js/modal.js

```diff
@@ -0,0 +1,25 @@
+const modalSection = document.getElementById('modal-section');
+// eslint-disable-next-line no-unused-vars
+function openModal() {
+  modalSection.classList.add('modal-section-display-on');
+}
+
+// eslint-disable-next-line no-unused-vars
+function updateModalInfo(title, text) {
+  document.getElementById('modal-title').innerText = title;
+  document.getElementById('modal-data-inputs').innerText = text;
+}
+
+function closeModal() {
+  modalSection.classList.remove('modal-section-display-on');
+  modalSection.classList.add('modal-display-off');
+}
+
+modalSection.onclick = (e) => {
+  if (e.target === modalSection) {
```

Comment:
is it necessary to check which target is?

Reply:
In this case it is, otherwise the modal would close if you click on any part of the modal (including text)